pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -642,12 +642,17 @@ func (a *ApplicationBackupController) handle(ctx context.Context, backup *stork_
 	case stork_api.ApplicationBackupStageApplications:
 		err := a.backupResources(backup)
 		if err != nil {
+			isLargeResourceError, err := utils.ReorganizeLargeResourceError(err)
 			message := fmt.Sprintf("Error backing up resources: %v", err)
 			log.ApplicationBackupLog(backup).Errorf(message)
 			a.recorder.Event(backup,
 				v1.EventTypeWarning,
 				string(stork_api.ApplicationBackupStatusFailed),
 				message)
+			// Check if the error is due to a large resource issue
+			if isLargeResourceError {
+				handleLargeResourceError(a, backup, message, maxRetry, retrySleep)
```

Comment:
Any reason for passing maxRetry and retrySleep to the function ?...Those are constants can be accessed within this file